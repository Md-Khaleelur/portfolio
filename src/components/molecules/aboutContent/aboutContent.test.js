import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import AboutContent from './aboutContent';

describe("AboutContent component",() =>{
    test("Matches the snapshot",() => {
        const aboutContent = create(<AboutContent/>);
        expect(aboutContent.toJSON()).toMatchSnapshot();

    })
    test("AboutContent is rendered",() => {
        const { getByTestId, getByText } = render(<AboutContent/>);
        expect(getByTestId("aboutContent")).toBeInTheDocument();
        
    });
   

});