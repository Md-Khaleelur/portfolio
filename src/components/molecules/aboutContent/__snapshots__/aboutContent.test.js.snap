// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutContent component Matches the snapshot 1`] = `
<div
  data-testid="aboutContent"
>
  <div
    data-testid="title"
  >
    <h2>
      About US
    </h2>
  </div>
  <div
    data-testid="para"
  >
    <p>
      About US page body content
    </p>
  </div>
</div>
`;
